### PR TITLE
[Realtek][RTL8721CSM] net/netmgr/ : RTL8721CSM loadable apps config bus fault fix

### PR DESCRIPTION
**Pull request by fengyunlai**
RTL8721CSM configs loopback disable. 1st commit for the modification of all defconfigs, fix netdev_lwip.c Macros, _netif_loop_output_ipv4 and _netif_loop_output_ipv6 are not defined if LWIP_HAVE_LOOPIF is 0.

### CODE REVIEW

File: os/net/netmgr/netdev_lwip.c [hunk 1]
```diff
@@ -617,12 +617,14 @@ static err_t _netif_loopif_init(struct netif *netif)
 
 	netif->name[0] = 'l';
 	netif->name[1] = 'o';
+#if LWIP_HAVE_LOOPIF
```
Comment (sangam-swami) [inline os/net/netmgr/netdev_lwip.c:620]: I think these common functionality is for loopback interface , why do we need to remove the condition?

**Review by sunghan-chang — DISMISSED** [review]

**Comment by sunghan-chang** [comment]
@fengyunlai Once @pillip8282 confirms, I will merge this.

**Review by sangam-swami — APPROVED** [review]
looks fine

File: os/net/netmgr/netdev_lwip.c [hunk 2]
```diff
@@ -668,7 +670,7 @@ static int lwip_init_nic(struct netdev *dev, struct nic_config *config)
 		return -1;
 	}
 
-	char *rnetif = (char *)malloc(sizeof(struct netif) + sizeof(struct netdev *));
+	char *rnetif = (char *)kmm_malloc(sizeof(struct netif) + sizeof(struct netdev *));
```
Comment (jeongchanKim) [inline os/net/netmgr/netdev_lwip.c:673]: Please change the `free` to `kmm_free` in line 733.

**Comment by sunghan-chang** [comment]
@fengyunlai There are more missing modification for malloc and free. How about doing by us?

**Comment by sunghan-chang** [comment]
Please find #4618

**Review by fengyunlai — COMMENTED** [review]
> @fengyunlai There are more missing modification for malloc and free. How about doing by us?

No problem, Mr Chang. Then I will revert my commits for "malloc to kmm_malloc" in this PR. Instead, I will fix the malloc/free usage of Realtek codes in this PR. How do you think?

**Comment by sunghan-chang** [comment]
> No problem, Mr Chang. Then I will revert my commits for "malloc to kmm_malloc" in this PR. Instead, I will fix the malloc/free usage of Realtek codes in this PR. How do you think?

That's good. Thank you @fengyunlai .

**Review by sunghan-chang — APPROVED** [review]

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**